oasislmf/computation/run/platform.py

```diff
@@ -135,24 +116,19 @@ def print_endpoint(self, attr, items):
     def select_id(self, msg, valid_ids):
         while True:
             try:
-                value = int(input(f'Select {msg} ID: '))
+                value = input(f'Select {msg} ID: ')
             except ValueError:
                 self.logger.info('Invalid Response: {}'.format(value))
                 continue
             except KeyboardInterrupt:
-                exit(1)
+                return -1
 
-            if (value < 0) or (str(value) not in valid_ids):
-                self.logger.info('Not a valid id from: {}'.format(valid_ids))
+            if str(value) not in valid_ids:
+                self.logger.info(f'Not a valid id from: {valid_ids} - ctrl-c to exit')
```

Comment:
I'd suggest a more informative:
```suggestion
            if value not in valid_ids:
                self.logger.info(f'id {value} not among the valid ids: {valid_ids} - ctrl-c to exit')
```